automation/jleechanorg_pr_automation/openai_automation/test_codex_comprehensive.py

```
@@ -0,0 +1,352 @@
+        automation.page.locator = Mock(return_value=mock_locator)
+
+        tasks = await automation.find_github_mention_tasks()
+        assert len(tasks) == expected_return
+        print(f"✅ Limit {limit} with {mock_task_count} tasks returned {expected_return}")
+
+    @requires_chrome
+    @pytest.mark.asyncio
+    async def test_default_limit_50_with_real_chrome(self):
+        """Test default limit of 50 with real Chrome instance."""
+        automation = CodexGitHubMentionsAutomation()
+        assert automation.task_limit == 50
```

Ruff (0.14.8)

Magic value used in comparison, consider replacing `50` with a constant variable

(PLR2004)